common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
I didn't find any alternatives short of writing our own functionality for finding all of the focusable and tab focusable elements on the page.  I did narrow this down to just the "selectable" portion of jquery.ui so that it is around 19 KB instead of 230 KB.